projects/demo/src/modules/components/block-status/block-status.template.html

```diff
@@ -43,6 +43,7 @@
         <tui-doc-demo>
             <tui-block-status [card]="card">
                 <img
+                    tuiSlot="top"
```

Comment:
Do we need to fix and add `tuiSlot` here?

<img width="431" alt="image" src="https://user-images.githubusercontent.com/12021443/236158983-bf4d2768-58d5-4cd6-8e08-ea72060f28e0.png">